lxd/firewall/drivers/drivers_util.go

```diff
@@ -0,0 +1,38 @@
+package drivers
+
+import (
+	"fmt"
+)
+
+// PortRangesFromSlice checks if adjacent indices in the given slice contain consecutive
+// numbers and returns a slice of port ranges ([startNumber, rangeSize]) accordingly.
+//
+// Note that if the input slice was parsed from multiple ranges (e.g. "80-81,5000") then
```

Comment:
@markylaing this part of the comment doesn't look right?

Reply:
Shall I just remove the note? If we wanted to keep track of user defined ranges we would need to return it when the address is parsed, so I wasn't sure if it was necessary to include this note or not in any case.

Reply:
I wasn't clear on what the comment actually meant.

Reply:
Ahh I see sorry. How about:

> This function cannot differentiate ranges from adjacent ports e.g. if the given slice is "[80,81,82]" then the returned
> range will be "80-82", regardless of whether the user input was parsed from "80-82" or "80,81,82".

Reply:
Oh I see, thats fine to keep the (clarified) comment in then.